packages/playground/website/builder/builder.js

```diff
@@ -0,0 +1,925 @@
+				const stepType = await getLastOfType(editor, 'step', event.end);
+				const properties = await getStepProperties(stepType);
+				const property = properties[prevKey[0]] ?? null;
+				const propType = property.type ?? null;
+				const propRef = property['$ref'];
+
+				if (propType === 'string') {
+					editor
+						.getSession()
+						.insert(
+							{ row: event.end.row, column: event.end.column },
+							' ""'
+						);
+					editor.moveCursorTo(event.end.row, 2 + event.end.column);
+				} else if (propRef === '#/definitions/FileReference') {
+					editor
+						.getSession()
+						.insert(
+							{ row: event.end.row, column: event.end.column },
+							' {}'
+						);
+					editor.moveCursorTo(event.end.row, 2 + event.end.column);
+				}
+			}
+		} else if (inserted === '[') {
+			editor
+				.getSession()
+				.insert(
+					{ row: event.end.row, column: event.start.column + 1 },
+					']'
+				);
+			return;
+		} else if (inserted === '{') {
+			editor
+				.getSession()
+				.insert(
+					{ row: event.end.row, column: event.start.column + 1 },
+					'}'
+				);
+			return;
+		} else if (inserted === ',') {
+			const nextIndent = (lines[1 + event.start.row].match(/^(\s+)/g) || [
+				'',
+			])[0];
+			if (nextIndent.length >= indent.length) {
+				return;
+			}
+			if (lines[event.start.row][-1 + event.start.column] !== '"') {
+				editor
+					.getSession()
+					.insert(
+						{ row: event.end.row, column: event.end.column },
+						'\n' + indent
+					);
+				editor.moveCursorTo(1 + event.end.row, 1 + indent.length);
+				return;
+			}
+			editor
+				.getSession()
+				.insert(
+					{ row: event.end.row, column: event.end.column },
+					'\n' + indent + '""'
+				);
+			editor.moveCursorTo(1 + event.end.row, 1 + indent.length);
+			editor.execCommand('startAutocomplete');
+		}
+	});
+
+	window.test = {
+		iframeSrc,
+		iframe,
+		textarea,
+		button,
+	};
+
+	button.addEventListener('click', () => {
+		try {
+			clearError();
+			runBlueprint(editor);
+		} catch (error) {
+			showError(error);
+		}
+	});
+
+	let prevWin;
+
+	newTab.addEventListener('click', () => {
+		runBlueprint(editor);
+		const query = new URLSearchParams();
+
+		query.set('mode', 'seamless');
+		// query.set('php', blueprint?.preferredVersions?.php);
+		// query.set('wp', blueprint?.preferredVersions?.wp);
+		const url =
+			`https://playground.wordpress.net/?${query}#` +
+			JSON.stringify(getCurrentBlueprint(editor));
+		if (prevWin) {
+			prevWin.close();
+		}
+		prevWin = window.open(url, '_blank');
+	});
+
+	if (window.location.hash) {
+		loadFromHash(editor);
+	} else {
+		formatJson(editor, {
+			landingPage: '/wp-admin/',
+			phpExtensionBundles: ['kitchen-sink'],
+			preferredVersions: {
+				php: '7.4',
+				wp: '5.9',
+			},
+			steps: [
+				{
+					step: 'login',
+					username: 'admin',
+					password: 'password',
+				},
+			],
+		});
+	}
+
+	runBlueprint(editor);
+
+	const editorState = {
```

Comment:
Let's fix this issue and any other ones in this PR:

> GitHub Actions / Lint and typecheck
> 'editorState' is assigned a value but never used

Reply:
@adamziel done: https://github.com/WordPress/wordpress-playground/pull/773/files#diff-21cb6ea89be95aa56dd4e93aada49e8ca1e7055ce64690acdf795e45fa7d44daR849